mds/authent/migrations/0005_pre_add_scope_restrictions_for_application_users.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.7 on 2019-11-26 09:32
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("authent", "0004_post_fill_default_aggregator_for"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="application",
+            name="users_allowed_scopes",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(max_length=32),
+                default=list,
```

Comment:
I think a nullable field will be better.An empty list could not mean the same as a null value, and a nullable field is preferable when adding it to an existing table. To run your migration, Postgres will have to lock the whole Application table, which may not be possible under heavy load on production.